docs/tutorial/graphical.rst

```diff
@@ -397,5 +397,51 @@ The estimates are automatically updated every 2 seconds.
 
+Flexible hiding of inputs
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There can be situations when it may be relevant to turn on or off a number of inputs (e.g. when a part of the measurement script is skipped upon turning of a single :code:`BooleanParameter`).
+For these cases, it is possible to assign a :code:`Parameter` to a controlling :code:`Parameter`, which will hide or show the :code:`Input` of the :code:`Parameter` depending on the value of the :code:`Parameter`.
+This is done with the :code:`group_by` key-word argument.
+
+.. code-block:: python
+
+    toggle = BooleanParameter("toggle", default=True)
+    param = FloatParameter('some parameter', group_by='toggle')
+
+When both the :code:`toggle` and :code:`param` are visible in the :code:`InputsWidget` (via :code:`inputs=['iterations', 'delay', 'seed']` as demonstrated above) one can control whether the input-field of :code:`param` is visible by checking and unchecking the checkbox of :code:`toggle`.
+By default, the group will be visible if the value of the :code:`group_by` :code:`Parameter` is :code:`True` (which is only relevant for a :code:`BooleanParameter`), but it is possible to specify other value as conditions using the :code:`group_condition` keyword argument.
+
+.. code-block:: python
+
+    iterations = IntegerParameter('Loop Iterations', default=100)
+    param = FloatParameter('some parameter', group_by='iterations', group_condition=99)
+
+Here the input of :code:`param` is only visible if :code:`iterations` has a value of 99.
+This works with any type of :code:`Parameter` as :code:`group_by` parameter.
+
+To allow for even more flexibility, it is also possible to pass a (lambda)function as a condition:
+
+.. code-block:: python
+
+    iterations = IntegerParameter('Loop Iterations', default=100)
+    param = FloatParameter('some parameter', group_by='iterations', group_condition=lambda v: 50 < v < 99)
+
+Now the input of :code:`param` is only shown if the value of :code:`iterations` is between 50 and 99.
+
+Using the :code:`hide_groups` keyword-argument of the :code:`ManagedWindow` you can choose between hiding the groups (:code:`hide_groups = True`) and disabling / graying-out the groups (:code:`hide_groups = False`).
+
+Finally, it is also possible to provide multiple parameters to the :code:`group_by` argument, in which case the input will only be visible if all of the conditions are true.
+Multiple parameters for grouping can either be passed as a dict of string: condition pairs, or as a list of strings, in which case the `group_conditions` can be either a single condition or a list of conditions:
+
+.. code-block:: python
+
+    iterations = IntegerParameter('Loop Iterations', default=100)
+    toggle = BooleanParameter('A checkbox')
+    param_A = FloatParameter('some parameter', group_by=['iterations', 'toggle'], group_condition=[lambda v: 50 < v < 99, True])
+    param_B = FloatParameter('some parameter', group_by={'iterations': lambda v: 50 < v < 99, 'toggle':, True]})
+
```

Comment:
Some minor errors in `param_B` example.
The part:
 - `group_by={'iterations': lambda v: 50 < v < 99, 'toggle':, True]}`
should be replaced by:
 - `group_by={'iterations': lambda v: 50 < v < 99, 'toggle': True}`

Reply:
Ahh yes indeed, thanks